thrust/cmake/ThrustHeaderTesting.cmake

```diff
@@ -148,6 +148,9 @@ function(thrust_add_header_test thrust_target label definitions)
       endif()
     endif()
 
+    # We are deprecating some thrust features, so disable deprecation warnings
+    target_compile_definitions(${headertest_target} PRIVATE CCCL_IGNORE_DEPRECATED_API)
```

Comment:
I think this should not be needed, since we have put the `CCCL_IGNORE_DEPRECATED_API` in the test for `thrust::tabulate_output_iterator`?

Reply:
This is for the header tests, where we synthesize a simple file with just the include and a main